### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by rushter**
- Using prints for logging is not very convenient. A lot of times people want to redirect them to text files. At least, logging should not output to stdin and use stderr instead. For example, I don't want to have debugging information and translation results in the same output. 
- [It's better ](https://stackoverflow.com/a/14247383/1202129)to use `is` when comparing to None
- Added GPU support

### CODE REVIEW

File: argostranslate/translate.py [hunk 1]
```diff
@@ -84,7 +86,6 @@ def combine_paragraphs(paragraphs):
 
         Args:
             paragraphs ([str]): A list of paragraphs.
-            num_hypotheses (int): Number of hypothetic results to be combined
```
Comment (PJ-Finlay) [inline argostranslate/translate.py:87]: Why is this removed?

File: argostranslate/translate.py [hunk 2]
```diff
@@ -375,8 +375,8 @@ def apply_packaged_translation(pkg, input_text, translator, num_hypotheses=4):
             else:
                 sbd_index = start_index + detected_sentence_index
             sentences.append(input_text[start_index:sbd_index])
-            logging.debug("start_index", start_index)
-            logging.debug("sbd_index", sbd_index)
+            logging.debug("start_index: %r", start_index)
+            logging.debug("sbd_index: %r", sbd_index)
```
Comment (PJ-Finlay) [inline argostranslate/translate.py:379]: f strings are preferred for string formatting

File: argostranslate/translate.py (hunk 1 above)
Comment (PJ-Finlay) [inline argostranslate/translate.py:87]: Why is this deleted?

Comment (rushter) [reply argostranslate/translate.py:87]: Because there is no such argument.

**Comment by PJ-Finlay** [comment]
This looks good thanks!

GPU support should be a pretty useful feature.

File: argostranslate/translate.py (hunk 1 above)
Comment (PJ-Finlay) [reply argostranslate/translate.py:87]: That's a good reason.
